### ISSUE TITLE
Can"t watch Livestreams on Youtube (also a few videos aren"t ) and i can"t watch Livestreams on twitch. 

### ISSUE THREAD

**Issue by DerSimonYT** [montoyo/webdisplays#16]
Hello. 
As the title said, i can"t watch livestreams on youtube and also a few videos not.
The livestreams on twitch aren"t working too.

I would like to hear if it"s possible to fix or not.
And why. I have heard that is is because of the "flash player "
I don"t know what this flash player means....
Thank you for your help.
In the minecraft version 1.7.10 it was still working, i could watch Livestreams on twitch with 5 other people.

Bye :) 💃

**Comment by Gamer95875** [comment]
flash player's what's used to open flash content but youtube hasn't been using flash for quite a while, (i can't remember what it uses) hopefully that answers your question.

**Comment by LeronDoesGaming** [comment]
I believe YouTube uses a HTML5 player. They migrated from Flash since HTML5 is supported for a lot of other browsers out there.

**Comment by montoyo** [comment]
Yeah it's not flash... neither YouTube nor Twitch use flash now... I'll try to update JCEF, see if it changes something.

**Comment by LordPhrozen** [comment]
yes please. want to watch twitch streams inside :)

**Comment by josh-65** [comment]
Has this had any updates tried. A lot of content i'm trying to watch does not work.
